frontends/web/src/routes/account/send/send.tsx

```diff
@@ -104,7 +103,7 @@ class Send extends Component<Props, State> {
     private utxos!: Component<UTXOsProps>;
     private selectedUTXOs: SelectedUTXO = {};
     private unsubscribe!: () => void;
-    private qrCodeReader: BrowserQRCodeReader = new BrowserQRCodeReader();
+    private qrCodeReader: any;
```

Comment:
@benma @KasparEtter FYI I didn't know how to keep the type here :(
but then this is only a POC of how to load bigger libraries